### PR TITLE
Solve some random typos

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

File: app/static/app/potree/workers/BinaryDecoderWorker.js [hunk 1]
```diff
@@ -270,7 +270,7 @@ Potree.Version.prototype.equalOrHigher = function(version){
 	}
 };
 
-Potree.Version.prototype.upTo = function(version){
+Potree.Version.prototype.up to = function(version){
```
Comment (pierotofy) [inline app/static/app/potree/workers/BinaryDecoderWorker.js]: This is actually a syntax error and will generate an error. We should reverse this change.

Comment (patrickelectric) [inline app/static/app/potree/workers/BinaryDecoderWorker.js]: Ooops

**Comment by patrickelectric** [comment]
@pierotofy done :)

**Comment by pierotofy** [comment]
Fantastic, thank you!

**Pull request merged**